### PR TITLE
AEM Migration Phase 2

### PR DESCRIPTION
**Pull request by prycrane**
[AEM Migration Phase 2](https://test.canada.ca/experimental/migration/aem-migration.html)

French pull request: [AEM Migration Phase 2 #150](https://github.com/canada-ca/systeme-conception/pull/150)

Preview: https://deploy-preview-253--design-system-canada-ca.netlify.app/about/

### Migrated pages

**About Canada.ca**
https://deploy-preview-253--design-system-canada-ca.netlify.app/about/

**Digital Transformation Office**
https://deploy-preview-253--design-system-canada-ca.netlify.app//about/digital-transformation-office.html

**Top tasks for Canada.ca**
https://deploy-preview-253--design-system-canada-ca.netlify.app/about/top-tasks-for-canada-ca.html

**Latest changes to the Canada.ca design system**
https://deploy-preview-253--design-system-canada-ca.netlify.app/about/latest-changes.html

**Canada.ca Content Style Guide**
https://deploy-preview-253--design-system-canada-ca.netlify.app/style-guide/

### CODE REVIEW

File: about/about.md
Comment (delisma) [inline about/about.md:1]: File name should be index.md, digital-transformation-office.md and top-tasks-for-canada-ca.md. Their urls would then read design.canada.ca/about, design.canada.ca/about/digital-transformation-office.html and design.canada.ca/about/top-tasks-for-canada-ca.html. I'm not sure if it's SEO friendlier but it feel more human readable.

Comment (prycrane) [reply about/about.md:1]: Sounds good

**Comment by donohuech** [comment]
@prycrane The Canada.ca Content Style Guide https://deploy-preview-253--design-system-canada-ca.netlify.app/style-guide/ seems to have the width constrained on it. The text is only going halfway across the page, whereas on the live version it goes across the whole width of the page (https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html).

**Comment by donohuech** [comment]
@prycrane Visited all 5 links to the migrated pages and clicked around other parts of the design system and everything looks good (just the one previous comment about width in Style Guide).

**Comment by prycrane** [comment]
> @prycrane The Canada.ca Content Style Guide https://deploy-preview-253--design-system-canada-ca.netlify.app/style-guide/ seems to have the width constrained on it. The text is only going halfway across the page, whereas on the live version it goes across the whole width of the page (https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html).

That's the new margin width.

**Comment by prycrane** [comment]
> @prycrane Visited all 5 links to the migrated pages and clicked around other parts of the design system and everything looks good (just the one previous comment about width in Style Guide).

Thanks!

**Review by prycrane — COMMENTED** [review]
Looks correct

**Pull request closed without merging**